src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -258,13 +258,21 @@ public OffenderDetailSummary getOffenderDetails(
         final @RequestParam(value = "contactTypes", required = false) Optional<List<String>> contactTypes,
         final @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) @RequestParam(value = "from", required = false) Optional<LocalDateTime> from,
         final @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) @RequestParam(value = "to", required = false) Optional<LocalDateTime> to,
-        final @RequestParam(value = "appointmentsOnly", required = false) Optional<Boolean> appointmentsOnly) {
+        final @RequestParam(value = "appointmentsOnly", required = false) Optional<Boolean> appointmentsOnly,
+        final @RequestParam(value = "eventId", required = false) Optional<Long> eventId,
```

Comment:
just be aware; for the public interface we tend to use `convictionId` rather than `eventId` in our ubiquitous language for this API.

Reply:
Thanks, I will amend